indico/web/forms/fields/datetime.py

```diff
@@ -179,9 +179,36 @@ class IndicoDateField(DateField):
     widget = JinjaWidget('forms/date_widget.html', single_line=True, single_kwargs=True)
 
     def __init__(self, *args, **kwargs):
+        self.allow_clear = kwargs.pop('allow_clear', True)
```

Comment:
Let's default to `False` here since we did not have a clear button before, and only make it clearable in `AttachmentPackageForm` and `VCRoomListFilterForm` - in all the other places the field is required.


Another option would be defaulting to `not self.flags.required` (if the flags from validators are available at this point - might have to do it after the super call)